app/controllers/v1/SearchController.scala

```diff
@@ -69,7 +78,8 @@ class SearchController @Inject() (ws: RequestGenerator) extends ControllerUtils
       val key = id.orElse(request.getQueryString("id")).getOrElse("")
       val res = period match {
         case x if x.length == fixedYeaMonthSize =>
-          search[UnitLinksListType](key, uriPathBuilder(sbrControlApiBase, key, Some(period)), periodParam = Some(period))
+          val uri = uriPathBuilder(sbrControlApiBase, key, Some(period))
+          search[UnitLinksListType](key, uri, periodParam = Some(period))
         case _ => BadRequest(errAsJson(BAD_REQUEST, "bad_request", s"Invalid date, try checking the length of given date $period")).future
```

Comment:
The messages should really be in a `conf/messages` file, but this can be done later on.